can/interfaces/pcan/pcan.py

```diff
@@ -64,7 +64,8 @@
 log = logging.getLogger("can.pcan")
 
 MIN_PCAN_API_VERSION = version.parse("4.2.0")
-
+IS_WINDOWS = PLATFORM == "Windows"
+IS_LINUX = PLATFORM == "Linux"
```

Comment:
We could also move these to `can/interfaces/pcan/basic.py` and just import `IS_WINDOWS` & `IS_LINUX`, and also change the if around `import winreg` in there to then use `if IS_WINDOWS` instead. Small thing tough.